resourcemanager/pool/workerpool/workerpool.go

```diff
@@ -42,10 +42,25 @@ type WorkerPool[T any] struct {
 	createWorker func() Worker[T]
```

Comment:
If you want a work pool which is used in special, not to the scheduler, you only skip it in the scheduler in the resource mange.

Reply:
```Tune(_ util.Component, pool util.GoroutinePool)``` 

In the scheduler, you can get the type of component and return hold to skip schedule.

Reply:
I suggest you can write your scheduler for your task. CPU scheduler is a toy.

Reply:
OK, we can implement the automatic tuning later.